src/fontstash.h

```diff
@@ -1041,6 +1058,18 @@ static FONSglyph* fons__getGlyph(FONScontext* stash, FONSfont* font, unsigned in
 	// Could not find glyph, create it.
 	scale = fons__tt_getPixelHeightScale(&font->font, size);
 	g = fons__tt_getGlyphIndex(&font->font, codepoint);
+	// Try to find the glyph in fallback fonts.
+	if (g == 0)
+	{
+		for (i = 0; i < font->nfallbacks; ++i)
+		{
+			fallbackGlyph = fons__getGlyph(stash, stash->fonts[font->fallbacks[i]], codepoint, isize, iblur);
+			if (fallbackGlyph != NULL)
```

Comment:
That's a bit weird. I tried to render only the emoji character using the same codebase with different backends. The codepoint of the emoji is `127881`. When I ran my app with `stb_truetype` backend, the glyph index returns `298`, and the emoji is rendered as expected. However, after I changed to `freetype` backend. I can still see the codepoint is `127811`, but the glyph index also returns `0` and never render the emoji. I have no idea why this happened.